test/parser/test.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * Node dependencies
+ */
+import fs from 'fs';
+import path from 'path';
+import { exec } from 'child_process';
+import peg from 'pegjs';
+
+/**
+ * External dependencies
+ */
+import { uniq } from 'lodash';
+import consoleTable from 'console.table';
+
+/**
+ * Module constants
+ */
+const grammarFile = path.join( __dirname, '../../blocks/api/post.pegjs' );
+const parserExecutable = process.env.PARSER;
+const fixturesDir = path.join( __dirname, 'fixtures' );
+const defaultParser = peg.generate( fs.readFileSync( grammarFile, 'utf8' ) );
+const fileBasenames = uniq(
+	fs.readdirSync( fixturesDir )
+		.filter( f => /(\.html|\.json)$/.test( f ) )
+		.map( f => f.replace( /\..+$/, '' ) )
+);
+const durations = [];
+
+describe( 'parser', () => {
+	fileBasenames.forEach( f => {
+		it( f, ( done ) => {
+			const inputFile = path.resolve( fixturesDir, f + '.html' );
+			const start = Date.now();
+			exec( `cat ${ inputFile } | ${ parserExecutable }`, ( err, parserOutput ) => {
+				const end = Date.now();
```

Comment:
This approach only counts the boot time for the `parserExecutable`. In the case of a script written with NodeJS, whatever you use inside your NodeJS file (native module, JS, or WASM) it makes almost no difference.

A difference is visible when you use another executable, like native Rust binary for instance.

I think these numbers can mislead the user. Should we remove them, and come with a more robust in-browser approach?